.github/commands.yml

```diff
@@ -1,12 +1,12 @@
 {
-  perform: false,
+  perform: true,
   commands: [
     {
       type: 'label',
-      name: 'duplicate',
-      allowTriggerByBot: true,
-      action: 'close',
-      comment: "Thanks for creating this issue! We figured it's covering the same as another one we already have. Thus, we closed this one as a duplicate. You can search for existing issues [here](https://aka.ms/vscodeissuesearch). See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.\n\nHappy Coding!"
+      name: 'Needs More Info',
+      allowTriggerByBot: false,
+      action: 'comment',
+      comment: 'Looks like we need for info to debug your particular issue. If you could attach your logs (ensure no private data is in them) to the issue it would help us fix the issue much faster.\n\nTo find your logs run the command `Developer: Open Logs Folder`, this will open the log file locally. We generally care most about renderer.log'
```

Comment:
`Looks like we need more info to debug...`

`this will open the folder containing the logs locally`

`renderer1.log`

But yeah - run this by the PMs or someone as well.